docs/blog/posts/2024/warp-2-0.md

```diff
@@ -0,0 +1,109 @@
+---
+title: '@warp-ds v2 - DBA Changes Are Here!'
+date: 2024-07-02
+---
+
+DBA beta changes are now available on NPM and EIK.
+
+---
+
+### How to Update Your Apps for DBA
+
+#### Packages to Install from NPM
+
+| Package Name       | Version        |
+| ------------------ | -------------- |
+| `@warp-ds/css`     | `2.0.0-next.1` |
```

Comment:
`2.0.0-next.1` is the one we have pointed v2 alias to, but now the current version on next is at `2.0.0-next.2` and once we make a stable release of drive (@warp-ds/uno), there will be a new PR in the css-repo that would then bump next-version to `2.0.0-next.3`. So, my question is, do we want it to say here `2.0.0-next.1` or `2.0.0-next.3`? :)